llvm/lib/Passes/PassRegistry.def

```diff
@@ -46,6 +46,7 @@ MODULE_PASS("attributor", AttributorPass())
 MODULE_PASS("annotation2metadata", Annotation2MetadataPass())
 MODULE_PASS("openmp-opt", OpenMPOptPass())
 MODULE_PASS("called-value-propagation", CalledValuePropagationPass())
+MODULE_PASS("ccudi", CancelCrossCUDebugInfo())
```

Comment:
please add reasonable name to the option, something like `cancel-cross-cu-debug-info`